infrastructure/main.tf

```diff
@@ -36,43 +36,9 @@ locals {
   }
 }
 
-data "google_compute_network" "existing-network" {
-  count   = var.create_network ? 0 : 1
-  name    = var.network_name
-  project = var.project_id
-}
-
-data "google_compute_subnetwork" "subnetwork" {
-  count   = var.create_network ? 0 : 1
-  name    = var.subnetwork_name
-  region  = var.subnetwork_region
-  project = var.project_id
-}
-
-module "custom-network" {
```

Comment:
why remove this? isn't there a create-network flag we can set instead?

that way from RAG you can set create-network = false and for other applications to true (if needed)

Reply:
recovered the change. 
Now in RAG `create-network` set to false.